.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     # Specify the execution environment. You can specify an image from Dockerhub or use one of our Convenience Images from CircleCI's Developer Hub.
     # See: https://circleci.com/docs/2.0/configuration-reference/#docker-machine-macos-windows-executor
     docker:
-      - image: cimg/node:16.14.0
+      - image: cimg/node:18.13.0
```

Comment:
this is probably fine but might be worth using the `node:18lts` tag so we can ensure we're always testing on the latest lts version.

Reply:
Good idea, alias for this is just `lts` https://circleci.com/developer/images/image/cimg/node#image-tags